### PR TITLE
Add Travis CI service

### LINKED ISSUE #8: Add a CI service
Given it is an open source project, is very important to see the status build.
Adding a simple badge and a free CI service would be enough

### PR DESCRIPTION
**Pull request by emancu**
Fixes #8

### CODE REVIEW

**Comment by emancu** [comment]
There is a test that will fail because it was fixed on #7

**Comment by emancu** [comment]
@jngli could you please check this PR ?

**Comment by jngli** [comment]
Looks good. I would do the merge once onboarding this project with CLA automation.  Please also take a look at the AI contribution guideline (https://github.com/Microsoft/AppInsights-Home/blob/master/CONTRIBUTING.md) and sign the contributor license if you have not done it yet.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: ruby
+rvm:
+  - 1.9.3
```
Comment (emancu) [inline .travis.yml]: @jngli we should remove `1.9.3` given is not supported anymore.

**Comment by emancu** [comment]
@jngli I already signed the contributor license =)

File: .travis.yml (hunk 1 above)
Comment (jngli) [inline .travis.yml]: The only version I'm running currently (in windows) is 1.9.3. I'll need to update to 2.0.0 then. Have you tested the other versions?

**Comment by jngli** [comment]
Good:) still need to wait for CLA notification.  Hopefully it's done soon.

**Comment by jngli** [comment]
We post the wrong contributor license, could you please sign at http://cla.microsoft.com?  Thanks!

**Comment by emancu** [comment]
@jngli done!

**Pull request merged**